lib/srv/reexec.go

```diff
@@ -213,6 +217,23 @@ func RunCommand() (errw io.Writer, code int, err error) {
 	if contfd == nil {
 		return errorWriter, teleport.RemoteCommandFailure, trace.BadParameter("continue pipe not found")
 	}
+	readyfd := os.NewFile(ReadyFile, fdName(ReadyFile))
+	if readyfd == nil {
+		return errorWriter, teleport.RemoteCommandFailure, trace.BadParameter("continue write pipe not found")
```

Comment:
I checked that, and if the ready fd is not closed, the parent process blocks forever. If this error or any other above is returned, the parent blocks. TBH I'm not sure how we can fix it. We could add a timeout, but I'm not sure how long the PAM can take if you use some network based plugins.